### DISCUSSION TITLE
My application is approved but I can't access GitHub Copilot.

### DISCUSSION THREAD

**Discussion by bahatarlaci** [community/community#144542] [GitHub Education]
### Hi everyone,

<img width="346" alt="image" src="https://github.com/user-attachments/assets/90cc2a39-5707-46f9-b9f6-622843f36598">

My application is approved but I can't access GitHub Copilot.

**Comment by davevad93** [comment]
Hi @bahatarlaci ,

When the benefits become available, the status label will turn purple and change to:

- `Congrats! Your request for @bahatarlaci was approved and your benefits are now available.`

The waiting time may vary depending on different factors, such as the volume of requests; it usually takes a few days, but sometimes it can take a little longer, even a couple of weeks.

For further reference you can look at these discussions: 
#132790 , #131665 , #132470 , #133531, #134279, #135849, #136887, #138721, #139268, #140287

And also the [Student Developer Pack Application FAQs](https://github.com/orgs/community/discussions/111352) :

- [How long after I submit my application will it be approved?](https://github.com/orgs/community/discussions/111352#how-long-after-i-submit-my-application-will-it-be-approved)
- [How long after I’ve been approved will I receive my academic benefits?](https://github.com/orgs/community/discussions/111352#how-long-after-ive-been-approved-will-i-receive-my-academic-benefits)
